src/styles/annotator/highlights.scss

```diff
@@ -1,35 +1,55 @@
 :root {
   // Default highlight styling configuration
   --hypothesis-highlight-color: rgba(255, 255, 60, 0.4);
+  --hypothesis-highlight-secondColor: rgba(206, 206, 60, 0.4);
```

Comment:
To check my understanding: the unconventional (camelCase) class names here are because these CSS variable names are generated dynamically by interpolating keys of a JS object? If so, it might be useful to add a comment to note this.

Reply:
Yeah, this is totally a dev ergonomic thing, both for key aesthetic and interpolation into CSS variable keys/values. I'll take another look at it; maybe it's dum.